src/main/java/sirius/biz/model/QueryController.java

```diff
@@ -124,8 +126,8 @@ public void query(WebContext webContext) {
 
                 UserContext.message(Message.info()
                                            .withTextMessage(Strings.apply("Showing %s of %s results - Query took %sms",
-                                                                          result.size(),
-                                                                          numberOfEntities,
+                                                                          Amount.of(result.size()).toRoundedString(),
+                                                                          Amount.of(numberOfEntities).toRoundedString(),
```

Comment:
NLS.toUserString could be used as well (?)

Reply:
That was my first try and didn't work

Reply:
Or do you mean wrapping it in an Optional and then calling toUserString on that?